samples/drivers/led_max7219/src/main.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2015 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <gpio.h>
+#include <spi.h>
+
+#define SPI_DEV  DT_SPI_1_NAME
+#define GPIO_DEV DT_GPIO_P0_DEV_NAME
+
+/* Any GPIO pin that suits your design */
+#define PIN_CS   18
+
+struct device *spi_dev;
+struct device *gpio_dev;
+
+struct spi_cs_control cs;
+
+void spi_init(void)
```

Comment:
Please don't mix functions and structure/variable definitions.

Reply:
Fixed